tests/dependency/test_local_dependencies_file_notebook.ipynb

```diff
@@ -103,11 +101,8 @@
         }
     ],
     "metadata": {
-        "interpreter": {
-            "hash": "36089b64e1fea64f368c2bdeda50d6646f8e8f431f4f5c23c53eea8a71ffad7d"
-        },
         "kernelspec": {
-            "display_name": "Python 3.9.9 64-bit ('substrafl': virtualenv)",
+            "display_name": "Python 3.8.1 ('subenv')",
```

Comment:
Why does this change ?

Reply:
I think it's the same reason it changed in one your PRs, the notebook was executed from my environment
not sure how to ensure we do not commit this, maybe we can update the pre-commit hook to remove this